src/SFA.DAS.AANHub.Api/Common/BaseRequestDetails.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.AANHub.Api.Common
+{
+    public class BaseRequestDetails
+    {
+        public string ActionName { get; set; } = null!;
+        public string ControllerName { get; set; } = null!;
+        public string? GetId { get; set; }
```

Comment:
Didn't see this being used?

Reply:
It was mainly being used in the GET logs, and one instance of the POST logs. I've removed the ID from the post log and removed this property.